### DISCUSSION TITLE
SML: hex to text?

### DISCUSSION THREAD

**Discussion by nikito7** [arendst/Tasmota#11532] [Ideas]
```
> 01 04 00 04 00 01 70 0b 
: 01 04 06 56 30 31 30 35 00 35 02 
```

56 30 31 30 35 00 = V0105

Is possible?

Or send all hex by mqtt?

**Comment by nikito7** [comment]
Maybe a XX to send raw bytes 

@gemu2015 ?

And decode in Home Assistent

I have bit strings too

We wil need one time commands....

Can I use sml() with modbus?

**Reply by gemu2015** [reply]
sorry don't understand your problem. please describe in detail what you want to accomplish

**Reply by nikito7** [reply]
Send hex bytes by mqtt

Id function size (hex bytes) crc  crc

**Reply by nikito7** [reply]
A new type like UUuu....

But all raw bytes

**Comment by nikito7** [comment]
@gemu2015 a decoder to send all hex bytes by mqtt
